### ISSUE TITLE
xcodes runtimes doesn't show bundled OS's on 14.1

### ISSUE THREAD

**Issue by MattKiazyk** [RobotsAndPencils/xcodes#242]
If my active(selected) Xcode is anything greater then 14+ the `xcodes runtime` call doesn't show a line with `Bundled with Xcode`

Steps to reproduce:
- Set your active to Xcode 14.1
- run command `xcodes runtime`

List _should_ show `iOS 16.1 (Bundled with Xcode)` but it doesn't 
```
-- iOS --
iOS 12.4
iOS 13.0
iOS 13.1
iOS 13.2.2
iOS 13.3
iOS 13.4
iOS 13.5
iOS 13.6
iOS 13.7
iOS 14.0.1
iOS 14.1
iOS 14.2
iOS 14.3
iOS 14.4
iOS 14.5
iOS 15.0
iOS 15.2
iOS 15.4
iOS 15.5
iOS 16.0
-- watchOS --
watchOS 6.0
watchOS 6.1.1
watchOS 6.2.1
watchOS 7.0
watchOS 7.1
watchOS 7.2
watchOS 7.4
watchOS 8.0
watchOS 8.3
watchOS 8.5
watchOS 9.0
watchOS 9.1 (Downloaded)
-- tvOS --
tvOS 12.4
tvOS 13.0
tvOS 13.2
tvOS 13.3
tvOS 13.4
tvOS 14.0
tvOS 14.2
tvOS 14.3
tvOS 14.4
tvOS 14.5
tvOS 15.0
tvOS 15.2
tvOS 15.4
tvOS 16.0
tvOS 16.1
```

It seems to work fine on Xcode 14.0.1 and 13.4.1, so perhaps something changed on

**Comment by StevenMagdy** [comment]
This bug happens because 16.1 doesn't exist yet in https://devimages-cdn.apple.com/downloads/xcode/simulators/index2.dvtdownloadableindex. Implemented a workaround in #230

**Comment by MattKiazyk** [comment]
Fixed in 1.2.0 - Thanks @StevenMagdy
